ci-operator/config/openshift/assisted-service/openshift-assisted-service-master__edge.yaml

```diff
@@ -65,6 +65,18 @@ images:
     openshift_release_golang-1.17:
       as:
       - registry.ci.openshift.org/openshift/release:golang-1.17
+  to: assisted-service-build-base
+- dockerfile_literal: |
+    FROM base-build
+    COPY . .
+  inputs:
+    assisted-service-build-base:
+      as:
+      - base-build
+    src:
+      paths:
+      - destination_dir: .
+        source_path: /go/src/github.com/openshift/assisted-service/.
```

Comment:
I wonder if that section would work directly in `assisted-service-build` without using an intermediate image

Reply:
yeah if what osher said is right we would have no other choice